Sources/TuistGenerator/Generator/ProjectGenerator.swift

```diff
@@ -236,4 +254,13 @@ final class ProjectGenerator: ProjectGenerating {
         try schemesGenerator.generateTargetSchemes(project: project,
                                                    generatedProject: generatedProject)
     }
+
+    private func determineProjectConstants() -> ProjectConstants {
+        // TODO:
+        //
+        // To maintain backwards compatibility with Xcode 10
+        // the Xcode10 constants are used unless the use of Xcode 11
+        // features are detected (e.g Swift PM dependencies)
+        return .xcode10
```

Comment:
Do you plan to read it using `xcode-select`?

Reply:
I was thinking of mimicking Xcode 11's behavior where it only bumps the version if you introduce swift packages otherwise keeps the older version. 

We'd need to add some logic as part of: https://github.com/tuist/tuist/pull/394

Reply:
xcode-select might make more sense as there might be more attributes which are not supported in Xcode 10 which have not yet discovered. But this should be followed up in a separate ticket.